front/pages/api/stripe/webhook.ts

```diff
@@ -1043,30 +1063,32 @@ async function handler(
                 { event },
                 "[Stripe Webhook] Received customer.subscription.deleted event with the subscription status = ended_backend_only. Ending the subscription without deleting any data"
               );
-              await matchingSubscription.update({
-                status: "ended",
-                endDate: new Date(),
-              });
+              await matchingSubscription.markAsEnded();
```

Comment:
This will put the subscription as "ended_backend_only" when we want it ended it since it comes from the webhook 🙏🏻  We need a parameter for `markAsEnded` 🙏🏻